src/redux/books/bookSlice.js

```diff
@@ -11,15 +11,17 @@ const bookSlice = createSlice({
   initialState,
   reducers: {
     addBook: (state, action) => {
-      state.books = action.payload;
+      state.books.push(
+        action.payload,
+      );
```

Comment:
- [OPTIONAL] I highly recommend not using the `push` method which can mutate your array. However, you can use the [Destructuring JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) method. You can find more [information here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) about it.

```suggestion
      state.books = [...state.books, action.payload];
```